vividus/src/main/java/org/vividus/runner/StepsPrinter.java

```diff
@@ -49,6 +51,8 @@ public class StepsPrinter
     private static final String DEPRECATED = "DEPRECATED";
     private static final String COMPOSITE = "COMPOSITE IN STEPS FILE";
     private static final String EMPTY = "";
+    private static final Pattern DEPRECATION_COMMENT_PATTERN = Pattern.compile("^!--\\h+Deprecated:\\h+.*",
+            Pattern.CASE_INSENSITIVE);
```

Comment:
I don't think it should be case insensitive, markers format should be strict: only upper case